packages/gatsby-plugin-gatsby-cloud/package.json

```diff
@@ -51,11 +51,11 @@
   },
   "sideEffects": false,
   "scripts": {
-    "build": "babel src --out-dir . --ignore \"**/__tests__\" && npm run clean && npm run copy-type-declarations",
+    "build": "babel src --out-dir . --ignore \"**/__tests__\" --extensions \".tsx,.ts,.js\" --source-maps && npm run clean && npm run copy-type-declarations",
```

Comment:
While you're at it you can also change the `--out-dir` to `dist` and then you don't need to edit the `.eslintignore` and it's more organized like in the other packages

Reply:
@LekoArts that's awesome! Has that always worked with Gatsby?

Reply:
@LekoArts great idea for the cleanup but the change to the build folder might break the imports for gatsbyjs.com [here](https://github.com/gatsby-inc/mansion/blob/master/cloud/gatsbyjs.com/src/dashboard/src/features/ContentLoader/index.tsx#L18). @veryspry suggested that we hold off until we can do it properly and do a version bump

Reply:
@TylerBarnes Yes, Gatsby always considers the `main` first. Other packages in our monorepo that are purely TS build their files into `dist` and point `main` to it.

@leithonenglish that's a bad practice on .com then 😅  The proper way to expose and use `components` would be to use the `exports` field on `package.json` then. https://nodejs.org/api/packages.html#package-entry-points

So could be something like:

```
  "main": "dist/index.js",
  "exports": {
    ".": "./dist/index.js",
    "./components": "./dist/components.js"
  },
```

That doesn't require a major bump.

Reply:
Another note here: Why were source maps activated? We normally don't emit those